Deploy/stacks/dynamic/stack-clients/src/main/java/com/cmclinnovations/stack/clients/postgis/PostGISClient.java

```diff
@@ -21,6 +21,11 @@ private Connection getDefaultConnection() throws SQLException {
     }
 
     private Connection getConnection(String database) throws SQLException {
+        try {
+            Class.forName("org.postgresql.Driver");
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException(e);
```

Comment:
It's good practice to add a message to the new exception explaining what has failed from the point of view of the current function.